src/controllers/project.ts

```diff
@@ -0,0 +1,13 @@
+import { FastifyRequest, FastifyReply } from "fastify";
+import { Project } from "../entities";
+
+export default async function project(
```

Comment:
[nitpick] - Why is this function is named just `project`? Is this some sort of convention you follow? It's a bit counter-intuitive to scatter around requests with files for me. The pattern that I'm familiar with is to couple endpoints' function feature/entity based on entities. In this case, I'd put `projects` function in this file and name it like `getProjects` and `getProjectById` for this one. If that makes sense 🤔

Reply:
Made a little research, and came across the repo for Node.js best practices. The pattern I mentioned above is so called 3 layers pattern (architecture), and it's not recommended anymore. What they suggest is to put boundaries between components(features) to make it more concerned with its feature. Our project is so small that we won't really need to be worried or strictly follow this pattern, but I still think making files for each API is a bit too aggressive segregation.

The rule: https://github.com/goldbergyoni/nodebestpractices/blob/master/sections/projectstructre/breakintcomponents.md

Reply:
lol, I am too used to GraphQL patterns. you are right. Let's decide which pattern would suit us. then we can update them.

Reply:
haha sure